src/components/MoneyReportHeader.tsx

```diff
@@ -980,6 +985,22 @@ function MoneyReportHeader({policy, report: moneyRequestReport, transactionThrea
                 danger
                 shouldEnableNewFocusManagement
             />
+            <ConfirmModal
+                title={translate('iou.deleteReport')}
+                isVisible={isDeleteReportModalVisible}
+                onConfirm={() => {
+                    setIsDeleteReportModalVisible(false);
+
+                    deleteAppReport(moneyRequestReport?.reportID);
```

Comment:
this line caused a bug - https://github.com/Expensify/App/issues/63094#issuecomment-2920500938